MetaMask/metamask-extension
# Bump snaps packages

## **Description**

Bump snaps packages to latest.

Summary of changes in the snaps deps:

- Fix a few issues with passing non-JSON-serializable values
- Block `wallet_revokePermissions`
- **Add `row` and `address` component**

This PR also adds functionality to support the addition of the `row` and `address` components.

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1570


## **Manual testing steps**

1. Go to https://metamask.github.io/snaps/test-snaps/latest
2. Install the transaction insight snap
3. Click the send transaction button on the dapp
4. Click the tx insight tab
5. See that the snap uses rows for its formatting

## **Screenshots/Recordings**

<img width="364" alt="Screenshot 2023-11-21 at 12 51 34" src="https://github.com/MetaMask/metamask-extension/assets/1561200/5360f85d-08a6-4a2d-85a8-c2e5331a78fa">

https://github.com/MetaMask/metamask-extension/assets/1561200/c2bcdbec-a9ac-47e4-9f5f-d097d696e955
